cypress/e2e/signIn.cy.js

```diff
@@ -29,6 +34,34 @@ describe('Sign In page', () => {
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    signInPage.visit();
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+    signUpPage.clickOkBtn();
+
+    homePage.assertHeaderDoesntContainUsername();
+  });
```

Comment:
The test for logging in with wrong credentials seems to be using the correct credentials (`user.email` and `user.password`). Consider using incorrect credentials to properly test this scenario.